interfaces/acados_matlab_octave/acados_template_mex/+acados_template_mex/generate_solver_matlab.m

```diff
@@ -1,93 +1,147 @@
 function generate_solver_matlab(acados_ocp_nlp_json_file)
 
-    acados_template_folder = [getenv('ACADOS_INSTALL_DIR'), '/interfaces/acados_template/acados_template'];
+    acados_template_folder = fullfile(getenv('ACADOS_INSTALL_DIR'),...
+                          'interfaces', 'acados_template', 'acados_template');
 
-    if ~isfile([getenv('ACADOS_INSTALL_DIR'), '/bin/t_renderer'])
-        error('acados/bin/t_renderer not found! Please download the t_renderer binaries from https://github.com/acados/tera_renderer/releases and place them in acados/bin (strip version and extension from the binary name)')
+    %% check if t_renderer is available -> download if not
+    if ispc()
+        t_renderer_location = fullfile(getenv('ACADOS_INSTALL_DIR'),'bin','t_renderer.exe');
+    else
+        t_renderer_location = fullfile(getenv('ACADOS_INSTALL_DIR'),'bin','t_renderer');
     end
+    
+    if ~isfile( t_renderer_location )
+
+        message = ['\nDear acados user, we could not find the binaries of the ',...
```

Comment:
Maybe change to "t_renderer binaries not found"? Why "the" in front of "t_renderer". We could also add the URL to the GitHub releases page in the message ;)